server/ColorDeconvolution/ColorDeconvolution.xml

```diff
@@ -60,5 +60,37 @@
       <longflag>stainColor_3</longflag>
       <default>0,0,0</default>
     </double-vector>
+    <region>
+      <name>region</name>
+      <label>Region</label>
+      <longflag>region</longflag>
+      <description>left,top,width,height of the region of interest.  All -1 means the whole image is used.</description>
+      <default>-1,-1,-1,-1</default>
+    </region>
+    <!-- And now the output parameters -->
+    <region>
+      <name>region</name>
+      <label>Region</label>
+      <channel>output</channel>
+      <description>left,top,width,height of the region of interest.  All -1 means the whole image was used.</description>
+    </region>
+    <string>
```

Comment:
Let just use the `file` type for these outputs and hand it over to Dave as it is still not clear what is the most convenient way for the front-end. I am hoping the write support in large_image will provide a way to associate it will location information.